libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -2288,6 +2287,27 @@ void ExpressionCompiler::appendCompareOperatorCode(Token _operator, Type const&
 				m_context << Instruction::SWAP1;
 				m_context << ((u256(1) << 32) - 1) << Instruction::AND;
 			}
+			else if (funType->kind() == FunctionType::Kind::External)
+			{
+				// TO DO::
+				// The stack will look like <fun1_addr> <fun1_selector> <fun2_addr> <fun2_selector>
+				// In order to compare you need to swap fun2_selector with fun1_addr to move fun1_addr up.
+				// You can use swap3 for that.
+				m_context << Instruction::SWAP3;
+				// now the stack looks like <fun2_selector> <fun1_selector> <fun2_address> <fun1_address>
```

Comment:
Can you reduce the number of these comments a bit?

Reply:
Yes definitely. These are merely for my reference till the main code gets done. I'll remove them